src/components/GraphCard/Graph.tsx

```diff
@@ -29,6 +29,7 @@ interface IGraphProps {
 const Graph: FC<IGraphProps> = ({data}) => {
 
     //store
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)